src/Form/Type/UiElement/RowType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' Rich Editor plugin for Sylius.
+ *
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusRichEditorPlugin\Form\Type\UiElement;
+
+use MonsieurBiz\SyliusRichEditorPlugin\Form\Type\RichEditorType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class RowType extends AbstractType
+{
+    /**
+     * @inheritdoc
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('content', RichEditorType::class, [
+                'required' => true,
+                'label' => 'monsieurbiz_richeditor_plugin.ui_element.monsieurbiz.row.field.content',
+                'constraints' => [
+                    new Assert\NotBlank([]),
+                ],
+            ])
+        ;
```

Comment:
**🎯 suggestion:** Maybe we can tag the Column element with `row-compatible` as example and filter this field by this tag only?‏

Reply:
Yes it can, but it's not that crazy to add, just some tags in a yaml and an option in the field in the FormType 🤭

Reply:
I prefer to separate it in another PR to not add complexity to this one.